ui/index.html

```diff
@@ -8,15 +8,18 @@
     </head>
     <body ng-controller="MainCtrl">
         <div id="global-container" class="d-flex flex-column h-100">
-            <div id="onscreen-overlay" ng-if="onscreenOverlayShowing"
-                 ng-click="setOnscreenShortcutsShowing(false)">
-               <div id="onscreen-shortcuts" ng-if="onscreenShortcutsShowing"
-                    stop-propagation-on-left-click>
```

Comment:
The removal of `stop-propagation-on-left-click` here means that the `stopPropagationOnLeftClick` directive is now unused, and can be deleted too:
https://github.com/mozilla/treeherder/blob/91500ef378ec71baa8cd03f451cb017829370e72/ui/js/directives/treeherder/main.js#L60-L73

(The removal of `stop-propagation-on-left-click` does mean that the behaviour has changed - in that people can click anywhere on the shortcuts to close them, rather than only around the edges or the close button, but I think that's a good change)